app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -153,7 +153,7 @@ class AamvaStateMaintenanceWindow
       ],
       'TN' => [
         # Last Sunday of every month from 11:00 pm Sunday to 2:00 am. Monday ET
-        { cron: '0 23 * * Sun#last', duration: 3 * 60 },
+        { cron: '0 23 * * Sun#last', duration_minutes: 3 * 60 },
```

Comment:
what if we create a `Data` struct to represent these, so instantiating these will error with misspelled/wrong keywords? Or a spec to make sure all the keys are correct?

Reply:
I'm working now on a spec to make sure the keys are correct. I'm going to do that as a follow-on PR so I don't hold anything up, though.

Reply:
Matt is working on a follow-up to do just that. This is change is here to get the deploy rolling again.

Reply:
☝️ jinx!